pkg/binary/envoytest/util.go

```diff
@@ -27,20 +27,17 @@ import (
 	"github.com/stretchr/testify/require"
 	"github.com/tetratelabs/log"
 
+	reference "github.com/tetratelabs/getenvoy/pkg"
 	"github.com/tetratelabs/getenvoy/pkg/binary"
 	"github.com/tetratelabs/getenvoy/pkg/binary/envoy"
 	"github.com/tetratelabs/getenvoy/pkg/manifest"
 )
 
-// reference indicates the default Envoy version to be used for testing.
-// This currently latest, but we may support a range at some point.
-var reference = "standard:1.17.1"
-
 var once sync.Once
 var errorFetchingEnvoy error
 
 // FetchEnvoyAndRun retrieves the Envoy indicated by reference only once. This is intended to be used with TestMain.
-// In CI, you can execute this to obviate latency during test runs: "go run cmd/getenvoy/main.go fetch standard:1.17.1"
+// Execute this to obviate latency during test runs: go run cmd/getenvoy/main.go fetch $(cat pkg/reference.txt)
 func FetchEnvoyAndRun(m *testing.M) {
```

Comment:
a later change will move tests that use this into the e2e thing and make this go away